test/k8sT/Nightly.go

```diff
@@ -159,3 +167,109 @@ var _ = Describe("NightlyK8sEpsMeasurement", func() {
 		}
 	})
 })
+
+var _ = Describe("NightlyK8sExamples", func() {
+
+	var kubectl *helpers.Kubectl
+	var logger *logrus.Entry
+	var initialized bool
+	var ciliumPath string
+	var demoPath string
+	var l3Policy string
+	var appService = "app1-service"
+
+	initialize := func() {
+		if initialized == true {
+			return
+		}
+		logger = log.WithFields(logrus.Fields{"testName": "NightlyK8sEpsMeasurement"})
+		logger.Info("Starting")
+
+		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+		kubectl.Delete(ciliumPath)
+
+		demoPath = fmt.Sprintf("%s/demo.yaml", kubectl.ManifestsPath())
+		l3Policy = fmt.Sprintf("%s/l3_l4_policy.yaml", kubectl.ManifestsPath())
+		initialized = true
+	}
+
+	BeforeEach(func() {
+		initialize()
+	})
+
+	AfterEach(func() {
+		kubectl.Delete(demoPath)
+		kubectl.Delete(l3Policy)
+	})
+
+	getAppPods := func() map[string]string {
```

Comment:
Can you add a comment saying what the values of these strings represent? I.e., maps [x] to [y]